### PR TITLE
add Pretty Shell

### PR DESCRIPTION
**Pull request by aerobounce**
[shfmt](https://github.com/mvdan/sh) wrapper plugin for Sublime Text 3.
In other words, Shell Script beautifier plugin for Sublime Text 3.

Surprisingly there is no shell script prettifier available to ST3 while Atom or VSCode has it.
Now here it is.

Recommended for scripting beginners, I've written this plugin for myself to learn how to write a script in correct syntax, and achieved how.

### CODE REVIEW

File: repository/p.json [hunk 1]
```diff
@@ -2191,6 +2191,18 @@
...
 			]
 		},
+		{
+			"name": "Pretty Shell",
+			"details": "https://github.com/aerobounce/Sublime-Pretty-Shell",
+			"labels": ["sh", "bash", "posix", "shell", "pretty", "format", "minify"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+										"platforms": ["osx", "linux"],
```
Comment (FichteFoll) [inline repository/p.json]: This line has too much indentation.

**Comment by FichteFoll** [comment]
Use the `tempfile` module to create temporary files safely and according to OS standards.

File: repository/p.json (hunk 1 above)
Comment (aerobounce) [reply repository/p.json]: Oops. Sorry for dumb mistake. Fixed.

**Comment by aerobounce** [comment]
Thanks for your review and the tip.
Fixed it and tagged as `1.0.1`. Additionally simplified codes.

**Comment by aerobounce** [comment]
Thanks!

**Pull request merged**